### PR TITLE
{Core} Reconstruction AzCommandGroup.

### PR DESCRIPTION
**Pull request by kairu-ms**
**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Core

**Comment by qwordy** [comment]
Could you fill in description section? Thanks

File: src/azure-cli-core/azure/cli/core/commands/command_operation.py [hunk 1]
```diff
@@ -72,7 +72,11 @@ def load_getter_op_arguments(self, getter_op, cmd_args=None):
     def apply_doc_string(self, handler):
         return self.ctx._apply_doc_string(handler, self.kwargs)  # pylint: disable=protected-access
 
-    def load_op_description(self, handler):
+    def load_op_description(self, handler=None):
+        if handler is None:
+            def handler():
+                """"""
```
Comment (qwordy) [inline src/azure-cli-core/azure/cli/core/commands/command_operation.py:78]:
```suggestion
```

Comment (kairu-ms) [reply src/azure-cli-core/azure/cli/core/commands/command_operation.py:78]: > 

If remove this line, the __doc__ property of handler cannot be updated in apply_doc_string

**Comment by kairu-ms** [comment]
> Could you fill in description section? Thanks

I will schedule a meeting for this RP and add description before that.

File: src/azure-cli-core/azure/cli/core/commands/command_operation.py (hunk 1 above)
Comment (kairu-ms) [reply src/azure-cli-core/azure/cli/core/commands/command_operation.py:78]: I will add comments in code later.

Comment (jiasli) [reply src/azure-cli-core/azure/cli/core/commands/command_operation.py:78]: How about adding some text as placeholder?

Comment (kairu-ms) [reply src/azure-cli-core/azure/cli/core/commands/command_operation.py:78]: > How about adding some text as a placeholder?

It's a good idea. But the placeholder text will be the default description of that handler.

**Pull request closed without merging**